kdoctor/src/commonMain/kotlin/org/jetbrains/kotlin/doctor/diagnostics/SystemDiagnostic.kt

```diff
@@ -8,8 +8,9 @@ class SystemDiagnostic : Diagnostic("System") {
         Message(
             if (System.type == SystemType.MacOS) ResultType.Success else ResultType.Failure,
             """
-                ${System.type} (${System.getVersion() ?: "N/A"})
+                OS: ${System.type} (${System.getVersion() ?: "N/A"})
                 ${System.getHardwareInfo() ?: ""}
+                Using Rosetta 2: ${if (System.isUsingRosetta) "Yes" else "No"}
```

Comment:
I think we should add a separate message with info level in case Rosetta is used and nothing if it is not.

BTW what are a possible consequences for the end user if rosetta is enabled? Are any changes in configuration required in that case?

Reply:
Agree, a separate info message makes more sense.
I thought it would be nice to know since at least you can't install anything using Homebrew under Rosetta. But as far as I remember, nothing too KMM-specific.